### PR TITLE
Update community-process.rst

### PR DESCRIPTION
**Pull request by lindy65**
Adding status types to UserVoice issues section

### CODE REVIEW

File: source/process/community-process.rst [hunk 1]
```diff
@@ -61,7 +61,15 @@ The various community channels are outlined below, along with the responsibiliti
 
     - Ops Team @mentions the Community PM with a list that includes issues:
 
-      - where status requires updating (Under Review, Planned, Started, etc.)
+      - where status requires updating
+      
+        - Under Review: Before a ticket is made (spec is needed)
+        - Planned: Jira ticket with a plan on how to build the feature
+        - Started: When it’s added to current sprint
+        - Accepting Pull Requests: Ticket that is accepting pull request
```
Comment (jasonblais) [inline source/process/community-process.rst]: can we change this to "help wanted"

File: source/process/community-process.rst [hunk 2]
```diff
@@ -61,7 +61,15 @@ The various community channels are outlined below, along with the responsibiliti
 
     - Ops Team @mentions the Community PM with a list that includes issues:
 
-      - where status requires updating (Under Review, Planned, Started, etc.)
+      - where status requires updating
+      
+        - Under Review: Before a ticket is made (spec is needed)
```
Comment (jasonblais) [inline source/process/community-process.rst]: Maybe after a ticket is made for it, but spec isn't completed.

That way there's a link community can follow

File: source/process/community-process.rst [hunk 3]
```diff
@@ -61,7 +61,15 @@ The various community channels are outlined below, along with the responsibiliti
...
+        - Planned: Jira ticket with a plan on how to build the feature
+        - Started: When it’s added to current sprint
+        - Accepting Pull Requests: Ticket that is accepting pull request
+        - Completed: When ticket is resolved
+        - Declined: When the suggestion does not fit with the product
```
Comment (jasonblais) [inline source/process/community-process.rst]: is there an example of when we've done so?

Comment (lindy65) [inline source/process/community-process.rst]: Yep, there are a few:
https://mattermost.uservoice.com/admin/v3/suggestions/10405716/activity
https://mattermost.uservoice.com/admin/v3/suggestions/16354759/activity
https://mattermost.uservoice.com/admin/v3/suggestions/13397460/activity

**Comment by jasonblais** [comment]
Minor update, otherwise looks good 👍

**Pull request merged**